x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/system/generic_file_details.tsx

```diff
@@ -183,7 +183,7 @@ export const SystemGenericFileDetails = pure<GenericDetailsProps>(({ data, conte
         outcome={outcome}
       />
       <EuiSpacer size="s" />
-      <NetflowRenderer data={data} />
+      <NetflowRenderer data={data} timelineId="test" />
```

Comment:
do you really want to be hardcoded here? I am saying that because this file is not a test file